### PR TITLE
refactor: replace query-string with URLSearchParams

### PR DESCRIPTION
**Pull request by visualfanatic**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [ ] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Issue Description
To reduce amount of dependencies we can use the built-in class called [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). 

### TODOs before merging

- [x] Add tests
- [x] Add changes to documentation (guides, repository pages, in-code descriptions)
- [x] A changelog entry is created automatically using the pull request title (do not manually add a changelog entry)

### CODE REVIEW

File: src/dashboard/Data/Browser/Browser.react.js [hunk 1]
```diff
@@ -501,7 +500,7 @@ class Browser extends DashboardView {
           if (name === 'objectId' || this.state.isUnique && name !== this.state.uniqueField) {
             return;
           }
-          if (!!required) {
```
Comment (mtrezza) [inline src/dashboard/Data/Browser/Browser.react.js:504]: Are these changes required for this PR?

**Comment by mtrezza** [comment]
@visualfanatic Could you merge master into this to resolve the conflicts?

File: src/dashboard/Data/Browser/Browser.react.js (hunk 1 above)
Comment (visualfanatic) [reply src/dashboard/Data/Browser/Browser.react.js:504]: @mtrezza No, my IDE run your ESLint config through this file, sorry! Would you accept a PR with changes from running ESLint through the codebase?

**Review by mtrezza — APPROVED** [review]
Looks good! Thanks for making Parse Dashboard 1 dependency lighter!

File: src/dashboard/Data/Browser/Browser.react.js (hunk 1 above)
Comment (mtrezza) [reply src/dashboard/Data/Browser/Browser.react.js]: Normally I would be happy about lint changes, but this looks suspicious, because a double negation has a purpose.

Comment (mtrezza) [reply src/dashboard/Data/Browser/Browser.react.js]: Thanks for reverting.

**Comment by parseplatformorg** [comment]
🎉 This pull request has been released in version [3.2.0](https://github.com/parse-community/parse-dashboard/releases/tag/3.2.0)
